etor/src/main/resources/transformation_definitions.json

```diff
@@ -132,6 +132,25 @@
           "args": { }
         }
       ]
+    },
+    {
+      "name": "ucsdOruObrOverrides",
+      "description": "Manual overwrites on OBR-4 fields for a specific code",
```

Comment:
I think we should specify in the description what OBR-4 values are being updated (OBR-4.3 and OBR-4.4)

Reply:
Updated the description to be more specific